classes/c.oeclass

```diff
@@ -22,13 +30,13 @@ SDK_CPPFLAGS		?= ""
 
 HOST_CPPFLAGS:native		 = "${BUILD_CPPFLAGS}"
 HOST_CPPFLAGS:cross		 = "${BUILD_CPPFLAGS}"
-HOST_CPPFLAGS:machine		 = "${MACHINE_CPPFLAGS}"
+HOST_CPPFLAGS:machine		 = "${MACHINE_CPPFLAGS} ${PIC_OPTION}"
```

Comment:
Again, adding PIC_OPTION to HOST_CPPFLAGS when it is based on TARGET architecture is wrong.

Maybe create both a HOST_PIC_OPTION and TARGET_PIC_OPTION, controlled by HOST_CPU_powerpc and TARGET_CPU_powerpc respectively (and in both cases USE_powerpc_32bit_addr).